source/databricks/tests/contract_utils.py

```diff
@@ -36,6 +36,21 @@ def assert_contract_matches_schema(contract_path, schema):
         assert expected_field["type"] == actual_field["type"]
 
 
+def assert_contract_matches_literal(contract_path, literal):
```

Comment:
Perhaps reverse the phrasing (it is actual that should match expected):

```suggestion
def assert_codelist_matches_contract(codelist, contract_path):
```